js/lesson2/tutorial.md

```diff
@@ -137,7 +137,7 @@ console.log('Apples and oranges are different: ' + notEqual);
 ```
 
 > You may also see `==` and `!=`, these are similar but have some quirks so it's generally recommended to avoid them.
-> You can just use `===` and `!==` and they will
+> You can simply use `===` and `!==` and they will
```

Comment:
@KimberleyCook not sure if `simply` is better than `just`. Maybe say `It's safest to use === and !== as they will always do what you expect`.